client/me/purchases/cancel-purchase/test/cancellation-effect.jsx

```diff
@@ -0,0 +1,136 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import useMockery from 'test/helpers/use-mockery';
+
+describe( 'cancellation-effect', function() {
+	const purchase = { domain: 'example.com' };
+	const purchases = {};
+	const productsValues = {};
+	let cancellationEffect;
+	let translate;
+
+	useMockery( mockery => {
+		mockery.registerMock( 'lib/purchases', purchases );
+		mockery.registerMock( 'lib/products-values', productsValues );
+	} );
+
+	beforeEach( function() {
+		purchases.getName = () => 'purchase name';
+		translate = ( text, args ) => ( { args, text } );
+		cancellationEffect = require( '../cancellation-effect' );
```

Comment:
You can use destructuring here:
```js
( { cancellationEffectHeadline, cancellationEffectDetail } = require( '../cancellation-effect' ) );
```
You just need to put parenthesis and declare both variables before 😄